screencap.py

```diff
@@ -7,15 +7,30 @@
 # WARNING: Any manual changes made to this file will be lost when pyuic5 is
 # run again.  Do not edit this file unless you know what you are doing.
 
-
 from PyQt5 import QtCore, QtGui, QtWidgets
 from PyQt5.QtGui import *
 from PyQt5.QtCore import *
 from PyQt5.QtWidgets import *
 import sys
 from PIL import ImageGrab
 import time
-from os import environ
+from os import environ 
+import os.path
+import json
+from datetime import datetime
+
+config = {
+  "image": {
+    "name": "screenshot",
+    "extension": ".png",
+    "path": " "
```

Comment:
I'd say that None is a better value here than an empty string.

```suggestion
    "path": None
```